exec_fd_test.c

```diff
@@ -0,0 +1,13 @@
+#include <common.h>
+#include <exec.h>
+
+int main() {
```

Comment:
I would love to, but this is still not enough! In order to do this elegantly, a getter function that looks up kernel flags ([such as this one](http://bxr.su/FreeBSD/sys/kern/kern_environment.c#451)) would be very helpful, and I am under impression that we are considering such interface and maybe even @mkaim is working on it (I honestly don't know). I would have implemented such launcher ages ago, but I abstain from it until the interface to kernel args ("environment") is ready.